lib/Doctrine/Sniffs/Spacing/EnsureSpaces.php

```diff
@@ -25,6 +8,9 @@
 
 trait EnsureSpaces
```

Comment:
We can move these methods to `Doctrine\Sniffs\Spacing\SpaceOnNotSniff`, which is the only class that uses it